Tweak default icewm preferences

Attention: this patch is applied in the %build phase, because it patches
a file generated by ./src/genpref. To edit the patch:

  quilt setup icewm.spec
  cd icewm-<version>
  quilt push -a
  ./configure <options from the spec>
  make -C src genpref
  ./src/genpref >src/preferences
  quilt import -p0 patches/icewm-preferences.patch
  quilt push
  vi lib/preferences
  quilt refresh
---
Index: b/src/preferences
===================================================================
--- a/src/preferences
+++ b/src/preferences
@@ -576,6 +576,7 @@
 
 #  Icon search path (colon separated)
 # IconPath="/usr/local/share/icons:/usr/local/share/pixmaps:/usr/share/icons:/usr/share/pixmaps:"
+IconPath="/usr/share/icons/Adwaita/16x16/legacy:/usr/share/icons/Adwaita/16x16/apps:/usr/share/icons/hicolor:/usr/share/icons:/usr/share/pixmaps"
 
 #  Colon separated icon theme list with wildcard support. Minus prefix - can be used to exclude themes.
 # IconThemes="*:-HighContrast"
@@ -594,6 +595,7 @@
 
 #  Command to lock display/screensaver
 # LockCommand=""
+LockCommand="xlock -mousemotion"
 
 #  Command to run on clock
 # ClockCommand="xclock -name icewm -title Clock"
@@ -617,9 +619,11 @@
 
 #  Command to shutdown the system
 # ShutdownCommand="test -e /run/systemd/system && systemctl poweroff"
+ShutdownCommand="/bin/sh -c 'systemctl poweroff ||:'"
 
 #  Command to reboot the system
 # RebootCommand="test -e /run/systemd/system && systemctl reboot"
+RebootCommand="/bin/sh -c 'systemctl reboot ||:'"
 
 #  Command to send the system to standby mode
 # SuspendCommand="test -e /run/systemd/system && systemctl suspend"
@@ -1353,15 +1357,18 @@ WorkspaceNames=" 1 ", " 2 ", " 3 ", " 4
 
 #  Display desktop background centered and not tiled
 # DesktopBackgroundCenter=0 # 0/1
+DesktopBackgroundCenter=0 # 0/1
 
 #  Support for semitransparent terminals like Eterm or gnome-terminal
 # SupportSemitransparency=1 # 0/1
 
 #  Resize desktop background to full screen
 # DesktopBackgroundScaled=0 # 0/1
+DesktopBackgroundScaled=1 # 0/1
 
 #  Desktop background image(s)
 # DesktopBackgroundImage=""
+DesktopBackgroundImage="BRANDING_PICTURE"
 
 #  Desktop background color(s)
 # DesktopBackgroundColor=""
